### PR TITLE
Gatewayfaq: for issue #13

### PR DESCRIPTION
**Pull request by KrishnaIyer**
- Added a new FAQ page
- Added a page with LED details

### CODE REVIEW

File: _content/gateways/faq/index.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+#### Q. What do the LED lights on the gateway indicate?
+
+The LED lights are a great way of understanding the current state of the gateway. Click [here](ledstatus.md) for the complete list.
+
+
+#### Q. What if I take the gateway to a different location. Do I have repeat the configuration?
+
+Once you configure the gateway, the settings are locked to that location. So, in case you move to a different location you will have to do the activation again. 
+
+Goto  (https://ttn.fyi/activate) and at Step 1, you can select your gateway from the list and configure them.
```
Comment (egourlao) [inline _content/gateways/faq/index.md]: > Goto

Typo

File: _content/gateways/faq/index.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+- Start a terminal session (putty: windows, terminal: macOS/linux) to UART with 115200@8N
+
+#### Q. What do the LED lights on the gateway indicate?
+
+The LED lights are a great way of understanding the current state of the gateway. Click [here](ledstatus.md) for the complete list.
+
+
+#### Q. What if I take the gateway to a different location. Do I have repeat the configuration?
+
+Once you configure the gateway, the settings are locked to that location. So, in case you move to a different location you will have to do the activation again.
```
Comment (egourlao) [inline _content/gateways/faq/index.md]: Can't you change a gateway's location in the gateway's console page, in Settings > Location?

File: _content/gateways/faq/index.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+Once you configure the gateway, the settings are locked to that location. So, in case you move to a different location you will have to do the activation again. 
+
+Goto  (https://ttn.fyi/activate) and at Step 1, you can select your gateway from the list and configure them.
+
+#### Q. How do I connect to the WiFi access point of the gateway?
+
+After going to  (https://ttn.fyi/activate) and registering your gateway, you will have to connect to the WiFi of the gateway and then connect to your home/office network. 
+
+Connect to the gateways WiFi access point ‘Things-Gateway-XXXX’ using the password thethings.
```
Comment (egourlao) [inline _content/gateways/faq/index.md]: You can use backticks (``) if you want here.

File: _content/gateways/faq/index.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+#### Q. How can I check if the gateway is working or not?
+
+To check the status of the gateway, goto the console as directed afterwards in the activation process. Under the gateway section (https://console.thethingsnetwork.org/gateways), you can see the status. If any device sends data through your gateway, you can check it under the ‘traffic’ section.
```
Comment (egourlao) [inline _content/gateways/faq/index.md]: > goto

Typo

File: _content/gateways/faq/index.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+#### Q. How can I reset my gateway?
+
+- Pressing the button shortly while powered on restarts the gateway.
+- Holding the button for 2 seconds while powered on it erased the WiFi settings.
+- Holding the button for 5 seconds while powered on it erases the WiFi settings and activation settings.
+- Holding the button while powering on it will erase the full serial flash
+
+#### Q. The second time I power my gateway, it takes time to start up. why?
+
+Upon every power cycle and every 24 hours, the gateway checks for new firmware. If the latest firmware is not downloaded yet it will download the new image and perform a reboot in order to install the image. This will take a few minutes. The process is indicated by a flashing power LED. Power cycling during this process is not dangerous, but this process will be restarted on the next start-up. So, please be patient
```
Comment (egourlao) [inline _content/gateways/faq/index.md]: Let's add missing punctuation: a period at the end, and a comma to make the second sentence lighter!

File: _content/gateways/faq/index.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+- First step - Connect the antenna and plug the gateway into a power outlet
+- Second step - Go to the website (https://ttn.fyi/activate)
+- Third step - Follow the instructions on the website
+
+
+#### Q. How can I reset my gateway?
+
+- Pressing the button shortly while powered on restarts the gateway.
+- Holding the button for 2 seconds while powered on it erased the WiFi settings.
```
Comment (egourlao) [inline _content/gateways/faq/index.md]: > erased

Typo

File: _content/gateways/faq/index.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+- Holding the button for 5 seconds while powered on it erases the WiFi settings and activation settings.
+- Holding the button while powering on it will erase the full serial flash
```
Comment (egourlao) [inline _content/gateways/faq/index.md]: Add a period at the end for coherence

File: _content/gateways/faq/index.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+#### Q. The second time I power my gateway, it takes time to start up. why?
+
+Upon every power cycle and every 24 hours, the gateway checks for new firmware. If the latest firmware is not downloaded yet it will download the new image and perform a reboot in order to install the image. This will take a few minutes. The process is indicated by a flashing power LED. Power cycling during this process is not dangerous, but this process will be restarted on the next start-up. So, please be patient
+
+#### Q. My gateway stays in AP mode. What do I do?
+
+This behaviour occurs when there is a glitch in the wifi connection of the gateway. When this happens, your gateway will turn into an AP(Access Point) and devices you used to set up the gateway can connect to it (your laptop or phone). To avoid this issue, you should ‘forget’ the wifi AP of the gateway on your phone/laptop.
```
Comment (egourlao) [inline _content/gateways/faq/index.md]: I think we can add a sentence here just to make more explicit what we mean by *forget*.

**Review by egourlao — CHANGES_REQUESTED** [review]
Great initiative!

As a general comment: you can use the [Markdown capabilities](https://daringfireball.net/projects/markdown/) more often in the documentation to format your text and make it extra readable.

File: _content/gateways/faq/index.md (hunk 2 above)
Comment (KrishnaIyer) [reply _content/gateways/faq/index.md]: absolutely right..

File: _content/gateways/faq/index.md [hunk 9]
```diff
@@ -58,18 +58,16 @@ The LED lights are a great way of understanding the current state of the gateway
 
 #### Q. What if I take the gateway to a different location. Do I have repeat the configuration?
 
-Once you configure the gateway, the settings are locked to that location. So, in case you move to a different location you will have to do the activation again. 
-
-Goto  (https://ttn.fyi/activate) and at Step 1, you can select your gateway from the list and configure them.
+No you don't have to re-configure. For better accuracy, it is recommeded to however update the location settings in the Console (Settings->Location).
```
Comment (egourlao) [inline _content/gateways/faq/index.md]: > recommeded

Typo + let's rephrase that question differently, e.g. "How do I change my gateway's location?"

**Review by egourlao — COMMENTED** [review]
Almost! Please ask for another review once you've addressed the points of a review in a new commit.

File: _content/gateways/faq/index.md (hunk 9 above)
Comment (KrishnaIyer) [reply _content/gateways/faq/index.md]: Q. What steps do I need to take if I want to relocate my gateway?

There is no need to re-configure it in this case. However, for better accuracy, it is recommended to update the location settings in the Console (Settings->Location).

Comment (egourlao) [reply _content/gateways/faq/index.md]: I think we can skip the reconfiguration sentence, for less confusion. For example:

> **If I change the location of my gateway, how can I change its location?**
> You can update the location settings in the Console. Go to your gateway's settings, and in the Location section, select the new location of your gateway.

**Review by egourlao — APPROVED** [review]

**Pull request merged**